common/lib/dependabot/pull_request_creator/github.rb

```diff
@@ -9,6 +9,9 @@ module Dependabot
   class PullRequestCreator
     # rubocop:disable Metrics/ClassLength
     class Github
+
+      MAX_PR_DESCRIPTION_LENGTH = 65536
```

Comment:
Since the constant declaration comes before `#create_pull_request`, you might want to attach the comment here instead.

Or, alternatively:

```suggestion
      MAX_PR_DESCRIPTION_LENGTH = 65_536 # characters (see #create_pull_request)
```